articles/node-vs-python/index.md

```diff
@@ -0,0 +1,179 @@
+
+Node.js has the upper hand here. Node.js is heavily optimized by Google's V8 JavaScript Engine, making it performant. Node.js follows an event loop based architecture, which makes it non-blocking. Asynchronous requests can be handled. This means that the runtime can execute multiple requests without getting blocked.
+
+#### Syntax and Ease of Use
+Python has extremely readable syntax. This makes it the preferred programming language for beginners to work on. It omits the use of curly braces and uses indentation to define scope.
+
+Node.js follows the JavaScript syntax, which is harder to understand than Python. But for programmers that are familiar with C/C++, Node.js syntax is similar and poses no challenge.
+
+#### Type of Projects
+Both Python and Node.js are used for web development and full-stack projects.
+
+Python is heavily used as a part of Data Science and Machine Learning projects. It has libraries like numpy, pandas, scikit, opencv and tensorflow that make it perfect for:
+
+- Neural Networks
+- Face Recognition
+- Data Processing
+
+Python is also preferred for small, lightweight websites. Flask, the micro-framework and Jinja, the templating engine is used by web developers.
+
+Due to the event loop architecture of Node.js, it is used for websites that involve a lot of concurrent processes. It is also suitable for Internet of Things solutions and projects.
+
+#### Frameworks and Expandability
+Python uses an interpreter called CPython that improves the performance of Python. There are many frameworks and libraries for full stack development.
+- [Django](https://www.djangoproject.com/) - A python web framework based on the Model-View-Controller Architecture
+- [Flask](https://flask.palletsprojects.com/en/1.1.x/) - A python micro-framework for web development
+- [Tensorflow](https://www.tensorflow.org/) - A python based Machine Learning platform
+
+Node.js has a ton of libraries that make it extensible, scalable and feature-rich.
+- [Express](https://expressjs.com/) - Unopinionated web framework for Node.js.
+- [Adonis](https://adonisjs.com/) - Full-Stack MVC Framework.
+- [Koa](https://koajs.com/) - Robust Foundation for Web Applications.
+
+#### Package Management
+Python uses a package manager called `pip`. It is extremely easy to use and has a ton of community-published packages. These packages can be found at the [Python Package Index](https://pypi.org/).
+
+It uses a file called **requirements.txt** to keep track of packages in a project.
+
+To install a package in python, just enter:
+
+```
+pip install <package-name>
+```
+
+Node.js uses a package manager called `npm`. The [npm registry](https://www.npmjs.com/) contains open source packages for Node.js, front-end web development, mobile apps and more.
+
+To keep track of packages in a project, Node.js uses a file called **package.json**
+
+To install a Node.js package, just type:
+
+```
+npm i <package-name>
+```
+
+### APIs in Node.js
```

Comment:
You should provide a brief explanation of what an API is and why you would want to build one. Also, you should provide your opinion (and stated as an opinion) on which of the two you would use and why (pros and cons of each) somewhere in this section.